### DISCUSSION TITLE
Textures will turn "Black"  in AR mode <model-viewer> using KTX 2.0 compression METHOD 3: ETC1S

### DISCUSSION THREAD

**Discussion by andygomez3dart** [KhronosGroup/KTX-Software#411] [Q&A] [answered]
Textures will turn "Black"  in AR mode <model-viewer> using KTX 2.0 compression METHOD 3: ETC1S

In my example I used the following textures:
Diffuse : JPG file,  2048x2048 pixels
Normal: PNG file, 1024x1024
glb size before compression : 10.9 MB
glb size after compression : 2.17 MB

Everything works great and fast in the viewport <model-viewer> when using a glb file with the METHOD 3: ETC1S.
The problem comes when I start the AR (augmented reality) mode and the 3D model turns "Black".

I wonder whether there is a problem with the way I applied the textures or it's just an incompatibility between <model-viewer> & KTX 2.0 compression at the moment.

Please find attached  the screenshots below:

Any idea about what it's going on?
![VID-20210502-WA0001](https://user-images.githubusercontent.com/79045928/116808796-a632e580-ab32-11eb-9b83-9f970d407b4f.gif)
![VID-20210502-WA0002](https://user-images.githubusercontent.com/79045928/116808797-aa5f0300-ab32-11eb-934b-c8692a0abae9.gif)

**Comment by MarkCallow** [comment, marked as answer]
This was answered in issue #409 which is a duplicate of this question.
